packages/xo-server/config.toml

```diff
@@ -4,6 +4,9 @@
 
 datadir = '/var/lib/xo-server/data'
 
+# Delay for which a resource is cached
+resourceDebounce = '5m'
```

Comment:
IMHO, this name is unclear, maybe `resourceCacheDelay`?

Reply:
It's named like that in the proxy, should i fix it also in the proxy?

Reply:
Not in this PR, I'll do it myself.

Do we agree on the name?

Reply:
I don't have any objection.